packages/core/src/shared/errors.ts

```diff
@@ -376,6 +377,12 @@ export function scrubNames(s: string, username?: string) {
         'Users',
         'users',
         'home',
+        'tmp',
+        'aws-toolkit-vscode',
+        'globalStorage', // from vscode globalStorageUri
+        CrashMonitoring.rootDir,
+        CrashMonitoring.runningExtDir,
+        CrashMonitoring.shutdownExtDir,
```

Comment:
is this going to un-scrub the user's entire /tmp dir path, or only the last segment?

Reply:
These are just the dir names, not the full path. As part of the move to the main constants file I've renamed these variables to be more clear.